Godeps/_workspace/src/k8s.io/kubernetes/pkg/apiserver/handlers.go

```diff
@@ -392,10 +392,16 @@ func WithAuthorizationCheck(handler http.Handler, getAttribs RequestAttributeGet
 	})
 }
 
-// APIRequestInfo holds information parsed from the http.Request
-type APIRequestInfo struct {
-	// Verb is the kube verb associated with the request, not the http verb.  This includes things like list and watch.
-	Verb       string
+// RequestInfo holds information parsed from the http.Request
+type RequestInfo struct {
+	// IsResourceRequest indicates whether or not the request is for an API resource
+	IsResourceRequest bool
+	// URL is the URL path of the request
+	URL string
```

Comment:
did this get updated with the latest from the upstream pull? seeing differences (Path vs URL)